api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionParseErrorsPaths.scala

```diff
@@ -36,22 +35,30 @@ trait SubmissionParseErrorsPaths
     path("filings" / Segment / "submissions" / IntNumber / "parseErrors") { (period, seqNr) =>
       timedGet { uri =>
         val supervisor = system.actorSelection("/user/supervisor")
-
         completeVerified(institutionId, period, seqNr, uri) {
-          val submissionID = SubmissionId(institutionId, period, seqNr)
-          val fHmdaFileParser = (supervisor ? FindProcessingActor(HmdaFileParser.name, submissionID)).mapTo[ActorRef]
-
-          val fHmdaFileParseState = for {
-            s <- fHmdaFileParser
-            xs <- (s ? GetState).mapTo[HmdaFileParseState]
-          } yield xs
-
-          onComplete(fHmdaFileParseState) {
-            case Success(state) =>
-              val summary = ParsingErrorSummary(state.tsParsingErrors, state.larParsingErrors)
-              complete(ToResponseMarshallable(summary))
-            case Failure(errors) =>
-              completeWithInternalError(uri, errors)
+          parameters("page".?) { (page: Option[String]) =>
+            val submissionID = SubmissionId(institutionId, period, seqNr)
+            val fHmdaFileParser = (supervisor ? FindProcessingActor(HmdaFileParser.name, submissionID)).mapTo[ActorRef]
+            val pageNum: Int = Try(page.getOrElse("").toInt).getOrElse(1)
```

Comment:
This might be done a little smoother with some of the built-in parameter functionality.

See the docs [here](http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/directives/parameter-directives/parameters.html), specifically the sections on optional parameters with defaults and deserialized parameters.